App.js

```diff
@@ -3,20 +3,32 @@ import { NavigationContainer } from "@react-navigation/native";
 import MainStack from "./routes/MainStack";
 import { useFonts } from "expo-font";
 import AppContext from "./store/App-Context";
+
+import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
```

Comment:
use [apollo-client/ApolloClient.js](https://github.com/bilalsha/delivery-fe/pull/2/files#diff-6b4c23094ea581974a131ccc7208a513d05ad4575fb12f8d4fe345317c0e04d4) class that you created to get client so that code is maintainable

Reply:
Done.